Models/AlphaBlendModeTest/README.body.md

```diff
@@ -40,20 +52,22 @@ Because the alpha values in the box are a smooth linear ramp, we can expect the
 
 Note that alpha blending should not be applied here:  Each texel is either fully opaque or discarded.  Depth buffering with writes enabled may be used in `MASK` mode.
 
-## Known Mipmapping Issues
+## Common problems with `MASK` and `alphaCutoff`
+
+### Known Mipmapping Issues
 
 ![screenshot](screenshot/MissingBorder.png)
 
 When a texture gets farther from the camera, a process called mipmapping may be used to blend texels (texturemap pixels) together.  This particular test model has a very thin opaque border adjacent to a lot of transparent texels.  When mipmapping blends these together, the resulting alpha values can land in the middle someplace.  Setting a high alpha mask cutoff value can cause these blended texels to be discarded.  This is not considered a failure for the purpose of this particular test.  However, if you see this effect in your own models, you may wish to set a lower cutoff value or increase the thickness of the affected area.
 
-## Problem: `alphaCutoff` Not Respected
+### Problem: `alphaCutoff` Not Respected
 
 ![screenshot](screenshot/CutoffValueFail.jpg)
 
 In the above screenshot, the requested `alphaCutoff` value is not being respected, and the default value of `0.5` appears to be applied regardless of the setting.  This is a test failure.
 
-## Problem: No Default Cutoff
+### Problem: No Default Cutoff
 
 ![screenshot](screenshot/CutoffDefaultFail.jpg)
 
-In this screenshot, it appears that the rendering engine is not using the value `0.5` for the default value of `alphaCutoff` when the model does not specify a particular value.  The glTF specification [names `0.5` as the default](https://github.com/KhronosGroup/glTF/blob/0890b76c62cc762ce82d4010df9bfebb1634839b/specification/2.0/schema/material.schema.json#L68), so this is a test failure.
\ No newline at end of file
+In this screenshot, it appears that the rendering engine is not using the value `0.5` for the default value of `alphaCutoff` when the model does not specify a particular value.  The glTF specification [names `0.5` as the default](https://github.com/KhronosGroup/glTF/blob/0890b76c62cc762ce82d4010df9bfebb1634839b/specification/2.0/schema/material.schema.json#L68), so this is a test failure.
```

Comment:
Please use the registry link here: https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#_material_alphacutoff.